python/lsst/ts/MTAirCompressor/aircompressor_csc.py

```diff
@@ -118,30 +52,45 @@ class MTAirCompressorCsc(salobj.BaseCsc):
     valid_simulation_modes: typing.Sequence[int] = (0, 1)
 
     def __init__(
-        self, index: int, initial_state=salobj.State.DISABLED, simulation_mode: int = 0
+        self,
+        index: int,
+        initial_state=salobj.State.DISABLED,
```

Comment:
DISABLED does not meet our CSC standards.

If you need to start this CSC in some other state than STANDBY then the usual solution is to enable specifying the desired initial state in the run_x command. I don't remember the details at the moment but would be happy to look it up. I think it just requires adding a single class constant.

Reply:
This was agreed with Tiago. Will go away in DM-35280